mindocr/data/predict_dataset.py

```diff
@@ -0,0 +1,80 @@
+'''
+Inference dataset class
+'''
+import os
+import random
+from typing import Union, List
+
+from .base_dataset import BaseDataset
+from .transforms.transforms_factory import create_transforms, run_transforms
+
+__all__ = ['PredictDataset']
+
+
+class PredictDataset(BaseDataset):
```

Comment:
I feel like `PredictDataset` should be a part of `DetDataset` rather than a separate class. Can set a flag such as `prediction` to load images only.